testfiles/AirflowNetwork_Simple_SmallOffice.idf

```diff
@@ -1456,9 +1456,6 @@
   Schedule:Compact,
     INF-SCHED,               !- Name
     Any Number,              !- Schedule Type Limits Name
-    Through: 12/31,          !- Field 1
```

Comment:
Through: 12/31 just needs to go at the end on both of these to get proper behavior.